src/source-pcap-file-helper.c

```diff
@@ -70,7 +70,7 @@ void PcapFileCallbackLoop(char *user, struct pcap_pkthdr *h, u_char *pkt)
 
     PKT_SET_SRC(p, PKT_SRC_WIRE);
     p->ts.tv_sec = h->ts.tv_sec;
-    p->ts.tv_usec = h->ts.tv_usec;
+    p->ts.tv_usec = h->ts.tv_usec % 1000000;
```

Comment:
can libpcap give us a timestamp like this?

Reply:
yes cf base64 encoded pcap : `1MOyoQIABAAAAAAAAAAAAP//AAABAAAAICAgICAgICAoAAAAKAAAACAgICAgICAgICAgIAgARSAAGiAgICAgICAgICAgICAgICAgICAgICA=`

libpcap gives 538976288 usec
Wireshark is also buggy